docs/concepts/integrating-with-github.md

```diff
@@ -22,6 +22,16 @@ Optionally, append a title and description to prepopulate the rule.
 
 ![capture example](../img/capture-github.png)
 
+### Reply
+
+```
+@CodeLingoBot [<your reply content>]
+```
+
+If CodeLingo finds an issue in your pull request, you can reply to the issue comment and your
```

Comment:
"Replying to a CodeLingo review comment whilst mentioning @CodeLingobot will add the reply to the corresponding rule discussion thread on the dashboard."

imo something like this is more concise and easier to understand